v2/emailpassword/common-customizations/signup-form/customising-each-form-field.mdx

```diff
@@ -221,7 +221,11 @@ For complex validations of fields, make use of [field validators](/docs/^{docsLi
 
 Currently, your sign-up form contains only email and password fields.
 While you can add additional simple input fields, the form also supports the integration of more sophisticated input types.
-These enhanced components can be radio buttons, checkboxes, dropdown, sliders, etc allowing for a more comprehensive user registration experience.
+These enhanced components include radio buttons, checkboxes, dropdowns, sliders, etc., allowing for a more comprehensive user registration experience.
+
+:::important
+You may need to disable the Shadow DOM if you're integrating with a different component library that requires you to import its own CSS. For instance, some component libraries, such as [react-international-phone](https://github.com/goveo/react-international-phone), might expect you to include their CSS alongside their components.
```

Comment:
Please tell them how to disable it as well.

Reply:
I provided a link to the article on disabling it.